### PR TITLE
Postex alt

### PR DESCRIPTION
**Pull request by CorneliaKelinske**
If you could just read through my About me page. 
I'm happy to change some stuff. but that's like the first version.

My next step will be to add pictures, so as to break up those text blocks.

And after that, I will write my first blog post.

Once that is done, I'd like to change the name of this repo/project and deploy

### CODE REVIEW

File: assets/css/custom.scss [hunk 1]
```diff
@@ -2,3 +2,33 @@
...
+.blog-author {
+  color: hsl(261, 31%,36%);
+}
+
+.message-header {background-color: hsl(261, 31%,36%);}
+
+
+
+
+.blog-body>h1 {
```
Comment (alanvardy) [inline assets/css/custom.scss]:
```suggestion
.blog-body > h1 {
```

File: assets/css/custom.scss [hunk 2]
```diff
@@ -2,3 +2,33 @@
...
+  font-size: 1.5rem;
+  font-weight: 600;
+  line-height: 1.125;  
+}
+
+html {
+  background-color: hsl(278, 7%, 83%);
```
Comment (alanvardy) [inline assets/css/custom.scss:32]: Nice, I like using hsl a lot.

File: lib/first_blog/blog.ex [hunk 3]
```diff
@@ -0,0 +1,7 @@
+defmodule FirstBlog.Blog do
+  @moduledoc "The blog context"
+  use Postex,
+    prefix: "https://conniecodes.com/posts/", # required
```
Comment (alanvardy) [inline lib/first_blog/blog.ex]:
```suggestion
    prefix: "https://connie.codes/posts/", # required
```

File: lib/first_blog_web/controllers/page_controller.ex [hunk 4]
```diff
@@ -2,14 +2,16 @@ defmodule FirstBlogWeb.PageController do
...
+    ]
+    render(conn, "index.html", assigns)
   end
 
   def about_me(conn, _params) do
-    render(conn, "about_me.html")
+    assigns = [
+      page_title: "About me"
+    ]
+    render(conn, "about_me.html", assigns)
```
Comment (alanvardy) [inline lib/first_blog_web/controllers/page_controller.ex:15]: You can just do

```suggestion
    render(conn, "about_me.html", page_title: "About me")
```

if that looks better to you

File: lib/first_blog_web/controllers/post_controller.ex [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  # alias FirstBlogWeb.ErrorView
+
+  @spec index(Plug.Conn.t(), map) :: Plug.Conn.t()
+  def index(conn, _params) do
+    assigns = [
+      page: 1,
+      posts: Blog.list_posts(1),
+      tags: Blog.tags_with_count(),
+      page_title: "Blog"
+    ] |> IO.inspect(label: "23", limit: :infinity, charlists: false)
```
Comment (alanvardy) [inline lib/first_blog_web/controllers/post_controller.ex]: IO.inspect.

Running `mix credo --strict` will probably catch this

Comment (alanvardy) [reply lib/first_blog_web/controllers/post_controller.ex]: We can quickly add continuous integration to this repo if you like too :+1:

File: lib/first_blog_web/templates/page/about_me.html.eex [hunk 6]
```diff
@@ -1,3 +1,63 @@
...
+
+
+    </div>
+
+    <div class="message-body">
+
+       <div class="block">My name is Cornelia Kelinske.</div>
+
+       <div class="block">Instead of listing all the things that I am and obsessing over what to put first, I will use a chronological approach,
+       showing how my interests and passions over time have led me to where I am at now.</div>
```
Comment (alanvardy) [inline lib/first_blog_web/templates/page/about_me.html.eex:16]: Programmers like the idea of a TLDR summary at the start. I think that could be really useful at the start.

Maybe give it different formatting to make it clear that it isn't part of the rest.

https://www.dictionary.com/browse/tldr

File: lib/first_blog_web/templates/page/about_me.html.eex [hunk 7]
```diff
@@ -1,3 +1,63 @@
...
+         </div>
+
+         <div class="block">And this is where I am at right now, living the dream. I enjoy hiking, swimming in the lake and in the ocean.Going for
+         bike rides with my family. I love having a nice glass of red wine while reading a good book in the hammock on my deck.
+         I also like having a big cup of coffee in the morning while I am journaling also in my hammock. And last but not least: I love lifting in
+         my garage and being a "bro". As for the second part of the word "BRO-GRAMMER": during the last two years, I have noticed a huge shift away from traditional translation
+         requests to more and more machine translation post editing jobs. Depending on the source text, those can be either wonderful because machine translation can be uncannily
+         good (for example for contracts) or extremely frustrating when jobs require specific terminology or when the source text contains errors (garbage in -> garbage out; or
+         as a friend once put it very accurately: "You can't turn excrement [she used the less formal term] into chocolate")</div>
```
Comment (alanvardy) [inline lib/first_blog_web/templates/page/about_me.html.eex:54]: I like it.

File: lib/first_blog_web/templates/page/about_me.html.eex [hunk 8]
```diff
@@ -1,3 +1,63 @@
...
+
+
+
+
+
+
+
+
+    </div>
```
Comment (alanvardy) [inline lib/first_blog_web/templates/page/about_me.html.eex:63]: What's with all the empty lines?

File: lib/first_blog_web/templates/post/index.html.eex [hunk 9]
```diff
@@ -0,0 +1,20 @@
+<h1 class=title><%= @page_title %></h1>
+
+<ul>
+  <%= for post <- @posts do %>
```
Comment (alanvardy) [inline lib/first_blog_web/templates/post/index.html.eex:4]: Some pages are 4 spaces indentation and others are 2

**Review by alanvardy — APPROVED** [review]

File: assets/css/custom.scss (hunk 1 above)
Comment (CorneliaKelinske) [reply assets/css/custom.scss]: Thanks!

File: lib/first_blog_web/controllers/page_controller.ex (hunk 4 above)
Comment (CorneliaKelinske) [reply lib/first_blog_web/controllers/page_controller.ex:15]: I actually deliberately used the assigns

File: lib/first_blog/blog.ex (hunk 3 above)
Comment (CorneliaKelinske) [reply lib/first_blog/blog.ex]: Yep, I guess I should change this

File: lib/first_blog_web/templates/page/about_me.html.eex (hunk 6 above)
Comment (CorneliaKelinske) [reply lib/first_blog_web/templates/page/about_me.html.eex:16]: Yeah, definitely something to think about

File: lib/first_blog_web/templates/page/about_me.html.eex (hunk 8 above)
Comment (CorneliaKelinske) [reply lib/first_blog_web/templates/page/about_me.html.eex:63]: That's just work in progress

File: lib/first_blog_web/templates/post/index.html.eex (hunk 9 above)
Comment (CorneliaKelinske) [reply lib/first_blog_web/templates/post/index.html.eex:4]: ? I don't know what you mean by this.... ?

**Comment by CorneliaKelinske** [comment]
@ alanvardy: thanks for all the improvements; however I do not understand your last comment there. Please clarify or show me in person. Thank you!

File: lib/first_blog_web/templates/post/index.html.eex (hunk 9 above)
Comment (alanvardy) [reply lib/first_blog_web/templates/post/index.html.eex:4]: Here are some examples

**Comment by CorneliaKelinske** [comment]
I think I changed it now...
But does it really show on the page?

**Pull request merged**